action.yml

```diff
@@ -1,13 +1,30 @@
-name: 'ggshield Local Action'
-description: 'Scan commits for security incidents (using repository image)'
+name: 'ggshield-action'
+description: 'Scan commits for hardcoded secrets and security policy breaks'
+author: GitGuardian <support@gitguardian.com>
+
+inputs:
+  args:
+    description: |
+      Arguments to be passed to ggshield secret scan
+      Options:
+        --show-secrets  Show secrets in plaintext instead of hiding them.
+        --exit-zero     Always return a 0 (non-error) status code, even if issues
+                        are found. The environment variable GITGUARDIAN_EXIT_ZERO can also be used
+                        to set this option.
+        --all-policies  Present fails of all policies (Filenames, FileExtensions,
```

Comment:
It may be on purpose : we are missing some options here. `ggshield secret scan --help` to get the full list.   
`--banlist-detector` could be of interest.

Reply:
@ziadghalleb, is this still open ? 🙂

Reply:
I can't close it from my account. I would like to start over at a later time.

Reply:
Oh, I can close it for you, but I'd be interested to know if you have a "Close pull request" button at the bottom of the page. Something like this:

![image](https://user-images.githubusercontent.com/91945295/177766436-9fde4a27-9412-4a53-bf31-4845e8348d50.png)

Reply:
Thanks. Unlike you, I don't see this button. It might have to do with the permissions I have on this repo (I am not a reviewer).

Reply:
OK, closing it for you then. It's odd GitHub does not let one close their own pull request.